processor/resourcedetectionprocessor/README.md

```diff
@@ -390,6 +390,26 @@ processors:
     override: false
 ```
 
+It also can optionally gather tags from the azure instance that the collector is running on.
+
+Azure custom configuration example:
+
+```yaml
+processors:
+  resourcedetection/azure:
+    detectors: ["azure"]
+    azure:
+      # A list of regex's to match tag keys to add as resource attributes can be specified
+      tags:
+        - ^tag1$
+        - ^tag2$
+        - ^label.*$
```

Comment:
Is just adding all tags acceptable in terms of performance and number? i.e. Do Azure instances typically have a lot of tags? 

Asking because I would like to keep it simple: if you want to filter, you can do that in a different component in your pipeline like the transform processor

Reply:
Azure supports a max of 50 tags per resource, how many are actually in use of course varies depending on people's use cases.
In terms of "cost", we essentially get the data for free since it is included in the resource's metadata query results, unlike on AWS which requires a secondary rest call to fetch the tag information. So we are only paying the cost of the regex matching. 

When implementing this I decided to keep it aligned with how the AWS implementation was exposed to users. The thought there was if someone is in a multi-cloud environment having them both configured the same, makes it easier for them to be used but I am not glued to that implementation choice.

Reply:
Thanks for the detailed reply! I agree it's valuable to be aligned with the AWS detector. I guess we can always add an 'add all of them' setting if needed.

Reply:
You're welcome. As for the `all` setting. Yup! In the interim, if people wanted to do this it would just require an unrestrictive regex `.*`

And CLA is finally signed. There was a hiccup on our side.